src/modules/control.cpp

```diff
@@ -2,60 +2,117 @@
 // Control Module
 //
 
+#include <stdint.h>
+
 #include "../pt/pt.h"
 
 #include "../hal/motor.h"
 #include "../hal/timer.h"
 
 #include "../modules/module.h"
 #include "../modules/control.h"
+#include "../modules/sensors.h"
 #include "../modules/parameters.h"
 
+#define INHALATION_OVERTIME 4 / 3
```

Comment:
This definition is a bit fragile in that it will not work correctly with enclosing parenthesis, i.e. `targetInhalationTime*(INHALATION_OVERTIME)`

I'd suggest turning this into a macro or a static inline function that forces the correct order of arithmetic operator evaluation.